### ISSUE TITLE
Make annotation querying more efficient

### ISSUE THREAD

**Issue by jdkent** [neurostuff/neurostore#149]
example output of an annotation:

```
{
    'dataset': '5NyJNxwSQq6M', 
    'notes': [{'study': '5FKuK2mnhCav', 'analysis': '8D4AZnoR3t5R', 'note': {'foo': 0}}, {'study': '7eoYBjxuzxhx', 'analysis': '3YS55PeVuNRL', 'note': {'foo': 1}}, {'study': '7eoYBjxuzxhx', 'analysis': '6sggvKsGAekm', 'note': {'foo': 2}}]
    'name': 'example annotation'
}
```
Takes a very long time to display annotations

alejandro's example of how to make a dumping more efficient.
https://github.com/neuroscout/neuroscout/blob/b818aae4e63c68e0051fa5eed9f15600a73ccb33/neuroscout/utils/db.py#L72-L83

**Comment by jdkent** [comment]
since connexion already validates responses, a low lying fruit would be to confirm that marshmallow is not also running validation and only serves as (de)serializer.

**Comment by jdkent** [comment]
closed by #239
